### PR TITLE
Created getDesksByState method

### PR DESCRIPTION
**Pull request by niker000**
1. Created getDesksByState method;
2. Added deskState field to DeskEntity;
3. Created tests on written method;
4. Some chenges in desk.csv and chengelog;

### CODE REVIEW

File: src/main/java/com/maybe/maybe/controller/EmployeeController.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+public class EmployeeController {
+
+    private EmployeeService employeeService;
+
+    public EmployeeController(EmployeeService employeeService) {
+        this.employeeService = employeeService;
+    }
+
+    @GetMapping("/employees/{id}")
+    public ResponseEntity<Employee> getEmployById(@PathVariable Long id) {
```
Comment (ghost) [inline src/main/java/com/maybe/maybe/controller/EmployeeController.java:25]: incorrect method name

File: src/main/java/com/maybe/maybe/dto/EmployeeDTO.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package com.maybe.maybe.dto;
+
+public class EmployeeDTO {
+
+    private String name;
+    private String login;
+    private String password;
+    private Integer roleId;
+
```
Comment (ghost) [inline src/main/java/com/maybe/maybe/dto/EmployeeDTO.java:9]: Where is fields validation?

File: src/main/java/com/maybe/maybe/dto/EmployeeRequest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
+package com.maybe.maybe.dto;
+
+public class EmployeeRequest {
+
+    private String name;
+    private String login;
+    private String password;
+    private Integer roleId;
+
```
Comment (ghost) [inline src/main/java/com/maybe/maybe/dto/EmployeeRequest.java:9]: Where is fields validation?

File: src/main/resources/db/changelog/changelog-master.xml [hunk 4]
```diff
@@ -5,7 +5,7 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <include file="db/changelog/changes/change_2018.xml"/>
+    <include file="db/changelog/changes/change_2019.xml"/>
     <include file="db/changelog/load/load-data.xml"/>
```
Comment (ghost) [inline src/main/resources/db/changelog/changelog-master.xml]: Did you pull from dev before? Surely these changes have already been in dev.

**Review by ghost — COMMENTED** [review]
You need to pull before this pull request

**Review by deivur — APPROVED** [review]

**Review by ArtemSobolenko — APPROVED** [review]

**Review by CaH4aZzz — APPROVED** [review]

**Review by kosten27 — APPROVED** [review]

**Pull request merged**